File: 5-Matemcatica/1212-Aritimetica_Primaria-8/1212.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        while(true){
            int e=0;
            int count1=0;
            
            string l1 = "";
            string l2 = "";
            
            string[] input = Console.ReadLine().Split(' ');
    
            if (input[0] == "0" && input[1]=="0"){
                break;
            }
            foreach(char id in input[0]){
                l1=l1+id;
            }
            foreach(char ik in input[1]){
                l2=l2+ik;
            }
            
            for(int i=0; i<9-input[0].Length; i++){
                l1=l1.Insert(0, "0");
            }
            for(int i=0; i<9-input[1].Length; i++){
                l2=l2.Insert(0, "0");
            }
            for (int j=8; j>-1; j--){
                
                int n1=int.Parse(Convert.ToString(l1[j]));
                int n2=int.Parse(Convert.ToString(l2[j]));
                if (n1+n2+e >9){
                    e=1;
                    count1+=1;
                
                }
                else{
                    e=0;
                }
            }
            if (count1==0){
                Console.WriteLine("No carry operation.");
            }
            
            else if (count1==1){
                Console.WriteLine("{0} carry operation.", count1);
            }
            else{
                Console.WriteLine("{0} carry operations.", count1);
            }
            
        }
    }
}

File: 5-Matemcatica/1212-Aritimetica_Primaria-8/1212.py
while True:    
    x=input().split()
    if x == ["0", "0"]:
        break
    l1=[]
    l2=[]
    e=0
    count1=0
    for i in x[0]:
        l1.append(int(i))
    for i in x[1]:
        l2.append(int(i))
    if len(l1)<9:
        for i in range(0, 9-len(l1)):
            l1.insert(0, 0)
    if len(l2)<9:
        for i in range(0, 9-len(l2)):
            l2.insert(0, 0)
    for i in range(9-1, -1, -1):
        n1=l1[i]
        n2=l2[i]
        if n1+n2+e >9:
            e=1
            count1+=1
        else:
            e=0
    if count1==0:
        count1="No"
    if count1==1 or count1=="No":
        print(f"{count1} carry operation.")
    else:
        print(f"{count1} carry operations.")